package.json

```diff
@@ -45,8 +45,6 @@
     "chartjs-plugin-doughnutlabel": "^2.0.3",
     "chartjs-plugin-labels": "^1.1.0",
     "chartjs-plugin-streaming": "^2.0.0",
-    "luxon": "^2.3.0",
-    "heatmap.js": "git+https://github.com/sanuradhag/heatmap.js.git",
```

Comment:
These need to be present or the tests will fail, along with probably some functionality.

Reply:
I've added them back. For whatever reason, I didn't see them in the merge conflicts or merge changes.